lambda/src/hyp3_floods.py

```diff
@@ -101,37 +97,34 @@ def process_active_hazard(hyp3: HyP3SubscriptionsAPI, hazard: dict, end: str) ->
     existing_subscription = get_existing_subscription(hyp3, name)
 
     if not existing_subscription:
-        print('No existing subscription; submitting new subscription')
+        print(f'No existing subscription; submitting new subscription with name: {name}')
         new_subscription = prepare_new_subscription(start, end, aoi, name)
         response = hyp3.submit_subscription(new_subscription)
         subscription_id = response['subscription']['subscription_id']
         print(f'Got subscription id: {subscription_id}')
     else:
-        compare_start_datetime(existing_subscription, start)
-        compare_aoi(existing_subscription, aoi)
-        subscription_id = existing_subscription['subscription_id']
-        print(f'Updating subscription with id: {subscription_id}')
-        hyp3.update_subscription(subscription_id, end)
+        log_updates(existing_subscription, start, aoi)
+        hyp3.update_subscription(
+            existing_subscription['subscription_id'],
```

Comment:
slightly prefer named parameters for all if using them for any
```suggestion
            subscription_id=existing_subscription['subscription_id'],
```